src/CTA.WebForms2Blazor/Factories/ClassConverterFactory.cs

```diff
@@ -73,7 +73,7 @@ public ClassConverter Build(string sourceFileRelativePath, SemanticModel model,
 
         public IEnumerable<ClassConverter> BuildMany(string sourceFileRelativePath, SemanticModel model)
         {
-            return model.SyntaxTree.GetNamespaceLevelTypes().Select(node => Build(sourceFileRelativePath, model, node));
+            return model.SyntaxTree.GetNamespaceLevelTypes().Select(node => Build(sourceFileRelativePath, model, node)).ToList();
```

Comment:
Null exception potentially here? If the model.SyntaxTree.GetNamespaceLevelTypes().Select(node => Build(sourceFileRelativePath, model, node)) resolves to null, if this is possible we should handle a null check here.

Reply:
I'm pretty sure `Select()` (and any Linq methods that return `IEnumerable<T>`) will never return null, if there are no results I'm pretty sure an empty enumerable is returned